editor/src/components/canvas/controls/insertion-plus-button.tsx

```diff
@@ -0,0 +1,249 @@
+import { ControlProps } from './new-canvas-controls'
+import * as React from 'react'
+import { MetadataUtils } from '../../../core/model/element-metadata-utils'
+import { colorTheme } from '../../../uuiui'
```

Comment:
All of these are functional components so they can use the `useColorTheme` hook instead

Reply:
Fixed in ae4e74f1.